WASC_2.00.sarif

```diff
@@ -0,0 +1,1111 @@
+{
```

Comment:
update the readme, add one example to the command following the others that i changed + add the link to the table to the file

Reply:
added.